Tasks/VsTest/vstest.ts

```diff
@@ -963,4 +939,59 @@ function isNullOrWhitespace(input) {
         return true;
     }
     return input.replace(/\s/g, '').length < 1;
-}
\ No newline at end of file
+}
+
+class VSTestVersion {
```

Comment:
Unless there's a perf penalty assocaiated, i would prefer this model for readability.